src/Service/Repository/SalesChannelRepository.php

```diff
@@ -44,9 +53,13 @@ public function __construct(
     public function getCurrentDomainUrl(SalesChannelContext $context): string
     {
         $criteria = new Criteria();
+        $domainUrlId = $this->configurationService->getDomainUrlId($context->getSalesChannelId());
+        $domainId = $this->configurationService->getDomainUrlId($context->getSalesChannelId()) ?:
```

Comment:
Can we make this config optional so that existing configurations are not broken?
```suggestion
        $domainId = $this->configurationService->getHreflangDefaultDomainId() ?:
```